catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -985,7 +992,8 @@ public CreateResponse create(CreateStorageRequest streamCreateRequest)
                                 e);
                     }
                 }
-
+                // TODO (RCZ) - pretty sure this if statement needs to check if content item has qualifier
```

Comment:
Yes it is a todo. Moreso, do the reviewers agree that this should check and make sure its setting it based off of the root contentItem (a non qualified one. eg, not the #preview, or #overview ), since those items will also be in the list of created content items and we probably do not want to use one of those uri's to set the metacard to. I made the change in latest update, if theres any that think this is not a bug, we can revert it.
done